website/src/pages/en/ai-suite/ai-introduction.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: The Graph's AI
+---
+
+Build with speed and scale faster with The Graph's AI Suite.
+
+## What is AI on The Graph?
+
+AI on The Graph connects large language models (LLMs) like Claude to onchain data indexed by The Graph. This allows developers to build AI systems that query blockchain data directly using natural language. This eliminates the need to write code or manage infrastructure.
+
+Instead of relying on static datasets or centralized APIs, you can now access live, verifiable blockchain data using [Graph Assistant](/ai-suite/graph-assistant/introduction/), [Subgraph MCP](), [Token API MCP]().
```

Comment:
There are two missing links here. If that's intentional (i.e. the pages are not ready?) then we should probably just remove the links.